agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/NetworkStatsbeatHttpPipelinePolicy.java

```diff
@@ -58,15 +58,16 @@ public NetworkStatsbeatHttpPipelinePolicy(NetworkStatsbeat networkStatsbeat) {
                 // these are not tracked as success or failure since they are just redirects
               } else if (statusCode == 402 || statusCode == 439) {
                 networkStatsbeat.incrementThrottlingCount(instrumentationKey, host);
+              } else if (StatusCodes.isRetryable(statusCode)) {
+                networkStatsbeat.incrementRetryCount(instrumentationKey, host);
```

Comment:
can we rename them in the code so it is more clear?
```suggestion
              } else if (StatusCodes.isRetryableFailure(statusCode)) {
                networkStatsbeat.incrementRetriableFailureCount(instrumentationKey, host);
```

Reply:
i still don't get it why it's not clear. isRetryable is to check if a status code is retryable? 
incrementRetryCount will keep track of number of time remote requests have been retried. 

most of the error code are type of failures... we don't say throttle failure, and simply throttle.. 

statsbeat network stats are tracking counts based on status code returned from breeze.

Reply:
good points, I think I agree, only confusion is really "retry count" sounds like the number of times we retried something, I think "retryable count" is clearer for what we are counting here

what do you think about just changing `incrementRetryCount` to `incrementRetryableCount`?

Reply:
i prefer to keep "incrementRetryCount" since that is the metric name we use for the sake of consistency.

Reply:
ok let's keep it as `incrementRetryCount` for now at least